### ISSUE TITLE
Double letters

### ISSUE THREAD

**Issue by mbtools** [hdegroot/zwordle#2]
There's a small bug on the selection screen.

If you use a letter twice in a guess and only one of the letters is correct (1 green, 1 black), then you enter the letter to "Black Letters" and to the correct letter position. See "R" in this example:

![image](https://user-images.githubusercontent.com/59966492/150522483-d55c8d8e-557a-40c5-bcdc-9e6364d9f2e6.png)

If you now press enter, then the "R" in position 2 is cleared:

![image](https://user-images.githubusercontent.com/59966492/150522617-bc8fa534-6910-4c8b-a6f3-6abf1eb82f18.png)

In other words, a letter can be black and correct.

**Comment by mbtools** [comment]
fixed in v2
